roles/kubernetes/node/tasks/main.yml

```diff
@@ -107,11 +107,11 @@
   tags:
     - kube-proxy
 
-- name: Modprobe nf_conntrack_ipv4
+- name: Modprobe nf_conntrack
```

Comment:
The commit 00e0f3bd2bf3ce4f83359c30e34545f32af20d20 shows
```
nf_conntrack is a dependency of ip_vs, so no need to care about it
```
So current nf_conntrack_ipv4 code is for older operating systems, and the lines of nf_conntrack have been intentionally removed.
I am not sure this change is valid according to the above commit.

/cc @champtar

Reply:
Thanks for the ping
if we want to change something we can make the `Modprobe nf_conntrack_ipv4` task never error using a `command` or `shell` and not `modprobe` + `ignore_errors`,
This PR as is breaks old OS support

Reply:
@oomichi is right.

I can confirm that nf_conntrack is a dependency of ip_vs, so it look like this modprobe task is designed for legacy OS and it has no effect on new OS anyway.

I wil close this PR. Many thanks to @champtar @oomichi